src/main/webapp/js/utils/table-utils.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const getTableColumn = (props) => {
+  const {
+    name, displayName,
+    filterable,
+  } = props;
+
+  const column = {
+    Header: displayName,
+    accessor: name,
+    filterable,
+    Cell: item => <div className="cell-test">{item.value}</div>,
```

Comment:
I think you should check the type here, the date fields will need to be formatted (the enums and relation too)
you can add a separate component for the Cell or just an utils function